src/nvim/path.c

```diff
@@ -2009,10 +2013,10 @@ int path_full_dir_name(char *directory, char *buffer, int len)
 
 // Append to_append to path with a slash in between.
 // Append to_append to path with a slash in between.
-int append_path(char *path, const char *to_append, int max_len)
+int append_path(char *path, const char *to_append, size_t max_len)
 {
-  int current_length = STRLEN(path);
-  int to_append_length = STRLEN(to_append);
+  size_t current_length = STRLEN(path);
```

Comment:
I didn't see the use of bare `strlen` in the TU so I thought that was a neovim standard to use the macro.